test/fuzz/fuzzer_pass_donate_modules_test.cpp

```diff
@@ -1678,6 +1679,135 @@ TEST(FuzzerPassDonateModulesTest, Miscellaneous1) {
   ASSERT_TRUE(IsValid(env, recipient_context.get()));
 }
 
+TEST(FuzzerPassDonateModulesTest, DonationSupportsOpTypeRuntimeArray) {
+  std::string donor_shader = R"(
+      ; SPIR-V
+      ; Version: 1.0
+      ; Generator: Google Clspv; 0
+      ; Bound: 50
+      ; Schema: 0
```

Comment:
Please remove these comment lines from the SPIR-V.

Reply:
Fixed in 29d7d77.